src/Canvas.tsx

```diff
@@ -339,8 +339,8 @@ const InternalCanvas: FC<CanvasProps & { ref?: Ref<CanvasRef> }> = forwardRef(
               }
             }}
           >
-            {layout?.edges?.map(renderEdge)}
             {layout?.children?.map(renderNode)}
+            {layout?.edges?.map(renderEdge)}
```

Comment:
Why did you move these? Just curious

Reply:
Because SVG uses ordering for whether things are in front. You cannot set z-index. Imo it makes more sense that edges are rendered on top, not behind nodes.

Reply:
Yup - i knew that - didn't know if there was a specific reason - I think I ordered them like this for a reason but I can't remember so fine w/ me.